### PR TITLE
Add check for stylus detection

### PR DESCRIPTION
**Pull request by stsdc**
This should prevent a crash of Settings, when Stylus in `null`.

### CODE REVIEW

File: src/Views/StylusView.vala [hunk 1]
```diff
@@ -31,6 +31,13 @@ public class Wacom.StylusView : Gtk.Box {
         append (stylus_box);
     }
 
+    public bool is_stylus_supported (Backend.WacomTool wacom_tool) {
+        debug ("%d", (int) wacom_tool.id);
+        unowned var stylus = wacom_db.get_stylus_for_id ((int) wacom_tool.id);
+        if (stylus == null) return false;
+        return true;
```
Comment (alainm23) [inline src/Views/StylusView.vala:38]: Evaluate Boolean directly
`return stylus != null;`

**Review by alainm23 — CHANGES_REQUESTED** [review]

**Review by alainm23 — APPROVED** [review]

**Comment by alainm23** [comment]
I don't have a Wacon to test, but at the coding level it's fine. Good work.

**Comment by alainm23** [comment]
It would be good if you update the metainfo file referencing the issue.

File: data/wacom.metainfo.xml.in [hunk 2]
```diff
@@ -31,6 +31,14 @@
...
 
   <releases>
+    <release version="8.0.1" date="2024-12-17" urgency="medium">
+      <description>
+        <ul>
+          <li>A fix for a crush that occured when stylus is not detected by libwacom</li>
+        </ul>
+      </description>
+    </release>
+
```
Comment (lenemter) [inline data/wacom.metainfo.xml.in:41]: You should put this into the `issues` tag:

```
    <release version="8.0.1" date="2024-12-17" urgency="medium">
      <description>
        <p>Updated translations</p>
      </description>
      <issues>
        <issue url="https://github.com/elementary/switchboard-plug-wacom/issues/69">Clicking on the wacom plug crushes The switchboard</issue>
      </issues>
    </release>
```

Comment (stsdc) [reply data/wacom.metainfo.xml.in]: Done.

File: data/wacom.metainfo.xml.in [hunk 3]
```diff
@@ -31,6 +31,15 @@
   <update_contact>contact_at_elementary.io</update_contact>
 
   <releases>
+    <release version="8.0.1" date="2024-12-17" urgency="medium">
+      <description>
+        <p>Updated translations</p>
+      </description>
+      <issues>
+        <issue url="https://github.com/elementary/switchboard-plug-wacom/issues/69">A fix for a crush of Settings that occured when stylus is not detected by libwacom.</issue>
```
Comment (alainm23) [inline data/wacom.metainfo.xml.in:39]:
```suggestion
        <issue url="https://github.com/elementary/switchboard-plug-wacom/issues/69">Clicking on the wacom plug crushes The switchboard</issue>
```

**Review by alainm23 — CHANGES_REQUESTED** [review]

File: data/wacom.metainfo.xml.in (hunk 3 above)
Comment (stsdc) [reply data/wacom.metainfo.xml.in]: Updated issue title and therefore the `<issue>` tag contents.

**Review by alainm23 — APPROVED** [review]

**Review by lenemter — APPROVED** [review]

**Pull request merged**